src/Point.ts

```diff
@@ -0,0 +1,30 @@
+export class Point {
+    readonly initX: number = 0;
+    readonly initY: number = 0;
+    private _x: number;
+    private _y: number;
+
+    constructor();
+    constructor(x: number, y: number);
+    constructor(x?: number, y?: number) {
```

Comment:
You can use default values
```suggestion
    constructor(x?: number = 0, y?: number = 0) {
```